index.bs

```diff
@@ -935,9 +954,13 @@ method is invoked, the user agent MUST:
             :   [=list/is not empty=]
             ::  1. Let |distinctTransports| be a new [=ordered set=].
 
-                1. If |allowCredentialDescriptorList| has exactly one value, let |savedCredentialId| be a new {{ArrayBuffer}},
-                       created using |global|'s [=%ArrayBuffer%=], and containing the bytes of
-                       <code>|allowCredentialDescriptorList|[0].id</code>.
+                1. If |allowCredentialDescriptorList| has exactly one value, let |savedCredentialId| be a new 
+                    {{PublicKeyCredentialDescriptor}}.{{PublicKeyCredentialDescriptor/id}} and set its value to <code>|allowCredentialDescriptorList|[0].id</code>'s
+                    value (see [here](#authenticatorGetAssertion-return-values) in [[#op-get-assertion]] for more information).
+
+                Issue: The foregoing step _may_ be incorrect, in that we are attempting to create |savedCredentialId|
```

Comment:
We'll see what others think, but this is clear enough to me to be implementable. Practically speaking, tracking the `CredentialId` for whatever credential works is going to be done by the hardware interaction code -- either keeping track of which ID worked on the wire or getting back the ID that came on the wire. So it's really a matter of plumbing it out of that hardware code, not even necessarily of this algorithm defining an internal slot to keep track of it.